x/concentrated-liquidity/position_test.go

```diff
@@ -16,6 +16,76 @@ import (
 
 var DefaultIncentiveRecords = []types.IncentiveRecord{incentiveRecordOne, incentiveRecordTwo, incentiveRecordThree, incentiveRecordFour}
 
+// AssertPositionsDoNotExist checks that the positions with the given IDs do not exist on the given accumulators.
+func (s *KeeperTestSuite) AssertPositionsDoNotExist(positionIds []uint64) {
+	uptimeAccumulators, err := s.clk.GetUptimeAccumulators(s.Ctx, defaultPoolId)
+	s.Require().NoError(err)
+
+	for _, positionId := range positionIds {
+		oldPositionName := string(types.KeyPositionId(positionId))
+		for _, uptimeAccum := range uptimeAccumulators {
+			// Check if the accumulator contains the position.
+			hasPosition, err := uptimeAccum.HasPosition(oldPositionName)
+			s.Require().NoError(err)
+			s.Require().False(hasPosition)
+		}
+
+		// Check that the old position has been deleted.
+		_, err := s.App.ConcentratedLiquidityKeeper.GetPosition(s.Ctx, positionId)
+		s.Require().Error(err)
+		s.Require().ErrorAs(err, &types.PositionIdNotFoundError{})
+	}
+}
```

Comment:
It says "on the given accumulators" but it seems like we only check uptime accumulators and not the fee accum. This is fine, but can we just be explicit about only checking the uptime accumulators in the godoc?

Reply:
This was leftover from an older/less general version of the abstraction – fixed now!